cli/yamls.py

```diff
@@ -15,95 +13,6 @@
 
 LOG = logger.LOG
 
```

Comment:
So, This removes the metaclass, and this module no longer needs to be imported on init, does it?
Also, consider renaming this module, or merging it with utils